src/arplot.py

```diff
@@ -210,10 +210,12 @@ def plot_orderfits(slf, model, ydata, xdata=None, xmodl=None, textplt="Order", m
                 xmn = np.min(xmodl)
                 xmx = np.max(xmodl)
             else:
-                xmn = min(np.min(xtmp), np.min(xmodl))
-                xmx = max(np.max(xtmp), np.max(xmodl))
+                xmn = np.min(xtmp)
+                xmx = np.max(xtmp)
+                #xmn = min(np.min(xtmp), np.min(xmodl))
+                #xmx = max(np.max(xtmp), np.max(xmodl))
```

Comment:
I was finding for LRIS that the extrapolation of the model (off the slit) was far from the data leading to a zoomed out fig.  Probably best is to take the model values where we have data.
Am doing so now.